.github/workflows/gemstash-ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.4, 2.5, 2.6, 2.7, jruby-9.2]
+        ruby: [2.4, 2.5, 2.6, 2.7, 3.0, jruby-9.2]
```

Comment:
Oh, this needs a quote

```suggestion
        ruby: [2.4, 2.5, 2.6, 2.7, "3.0", jruby-9.2]
```

This is a very small change, but which avoids a Float-to-String loss of characters.

An example (not from this project) of what it can look like before this change, in this picture:

<img width="376" alt="bild" src="https://user-images.githubusercontent.com/211/119782378-6ad6cd00-becc-11eb-9e13-9a425adaa054.png">

Read more details, if you like: https://github.com/actions/runner/issues/849

Reply:
<img width="461" alt="bild" src="https://user-images.githubusercontent.com/211/141267843-10eff6d6-90f6-46a7-8550-4ec7b0c74d6b.png">

Here is the output from this run, of this PR.

Reply:
By quoting, we'd see "3.0", there.

Reply:
@olleolleolle Thanks for the reminder  :)